src/genome/body.cpp

```diff
@@ -0,0 +1,161 @@
+//
+// Created by Emmet on 11/10/2022.
+//
+
+#include "body.h"
+
+BodyBuilder::BodyBuilder() :
+    currentX(0), currentY(0), currentDirection(Direction::RIGHT), anchors() {}
```

Comment:
Is current direction the way that the body is facing, or the way that it is heading? Should this be randomly assigned instead of hardcoded as always going to the right?